spark-jobs/src/main/scala/filodb/downsampler/chunk/DownsamplerMain.scala

```diff
@@ -205,11 +200,14 @@ class Downsampler(settings: DownsamplerSettings) extends Serializable {
 
     if (settings.exportIsEnabled && settings.exportKeyToRules.nonEmpty) {
       val exportKeys = settings.exportKeyToRules.keys.toSeq
+      val exportTableConfigs = settings.exportKeyToRules.values.toSeq
```

Comment:
Please make sure these orders are consistent.

Reply:
Created a seq containing pairs of (key, val) to be used further by exportKey(). Changes implemented in the latest commit.